modules/promotion/config/schema/commerce_promotion.schema.yml

```diff
@@ -0,0 +1,21 @@
+commerce_checkout.commerce_checkout_pane.coupon_redemption:
+  type: commerce_checkout_pane_configuration
+  mapping:
+    title:
+      type: string
+      label: 'Element title'
+    description:
+      type: string
+      label: 'Element description'
+    submit_title:
+      type: string
+      label: 'Submit button title'
+    submit_message:
+      type: string
+      label: 'Submit message'
+    remove_title:
+      type: string
+      label: 'Remove button title'
+    single_coupon_mode:
```

Comment:
I don't like this wording. "use_multiple"? It's the same wording used in the element type.